components/x-podcast-launchers/src/map-concept-to-acast-series.js

```diff
@@ -1,10 +1,9 @@
 import { brand } from '@financial-times/n-concept-ids';
 
+const acastSeriesMap = new Map([
```

Comment:
@fenglish I've tweaked this slightly so we export the Map itself. Think it makes this module a bit clearer

Reply:
@robsquires Thank you for doing that and also thank you for merging this PR.